appOPHD/UI/FileIo.cpp

```diff
@@ -19,8 +19,14 @@ using namespace NAS2D;
 
 
 FileIo::FileIo(FileLoadDelegate fileLoadDelegate) :
+	FileIo{fileLoadDelegate, FileSaveDelegate{}}
+{}
+
+
+FileIo::FileIo(FileLoadDelegate fileLoadDelegate, FileSaveDelegate fileSaveDelegate) :
 	Window{"File I/O"},
 	mFileLoadDelegate{fileLoadDelegate},
+	mFileSaveDelegate{fileSaveDelegate},
```

Comment:
Probably makes sense to add a call to the new delegate in this commit, so the new feature is usable. It would need to be a checked call, since a `Delegate` is default initialized to `nullptr` values. Additionally, we don't always allow saving, such as when `FileIo` is opened from the main menu. Though it would never be activated with a "Save" action in that case.